golem-cli/src/diagnose.rs

```diff
@@ -170,57 +171,50 @@ impl SelectedLanguage {
 impl Language {
     pub fn tools(&self) -> Vec<Tool> {
         match self {
-            Language::CCcp => vec![Tool::WasiSdk, Tool::WitBindgen, Tool::WasmTools],
-            Language::Go => vec![
-                Tool::TinyGo,
+            Language::CCcp => vec![
+                Tool::WasiSdk,
                 Tool::WitBindgen,
                 Tool::WasmTools,
-                Tool::GolemSdkGo,
+                Tool::CMake,
             ],
+            Language::Go => vec![Tool::Go, Tool::TinyGo, Tool::WasmTools, Tool::GolemSdkGo],
             Language::JsTs => vec![
                 Tool::Npm,
                 Tool::Jco,
                 Tool::ComponentizeJs,
                 Tool::GolemSdkTypeScript,
             ],
-            Language::Python => vec![Tool::ComponentizePy],
+            Language::Python => vec![Tool::ComponentizePy, Tool::Uv],
```

Comment:
i _think_ uv should manage comp-py too, so it should not be available globally

Reply:
so we probably should test through uv

Reply:
yes, exactly :+1:

Reply:
what does "test through uv" mean?

Reply:
`uv run componentize-py --version`
and that only works _in the component_ directory, not in the project root

Reply:
and this also means that the dep order should be Uv, CompPy